vernac/comInductive.mli

```diff
@@ -19,9 +19,14 @@ open Decl_kinds
 
 (** Entry points for the vernacular commands Inductive and CoInductive *)
 
+type uniform_inductive_flag =
+  | UniformParameters
+  | NonUniformParameters
+
 val do_mutual_inductive :
   (one_inductive_expr * decl_notation list) list -> cumulative_inductive_flag ->
-  polymorphic -> private_flag -> Declarations.recursivity_kind -> unit
+  polymorphic -> private_flag -> uniform_inductive_flag ->
+  Declarations.recursivity_kind -> unit
```

Comment:
Might be worth giving a label to this argument. The other ones are lacking it, but it doesn't justify not doing it systematically when introducing new arguments.